contracts/l1/Bridge.sol

```diff
@@ -79,6 +89,12 @@ contract Bridge is IBridge, VersionedInitializable {
         uint16 referralCode,
         bool fromUnderlyingAsset
     ) external override onlyValidL2Address(l2Recipient) returns (uint256) {
+        require(
+            IScaledBalanceToken(l1AToken).scaledBalanceOf(address(this)) +
```

Comment:
This is to not take into account the interest growth, right? Quite important to document this correctly on the `initialize()`

Reply:
correct!